db/seeds.rb

```diff
@@ -9,7 +9,7 @@
   %w(test1@test.com test2@test.com).each_with_index do |email, index|
     User.find_or_create_by!(email: email) do |user|
       user.skip_confirmation!
-      user.reset_password('password', 'password')
+      user.reset_password('a long password!', 'a long password!')
```

Comment:
May I suggest leaving off the exclamation point? I don't know about other devs, but I sign in with the test accounts all the time, and I'd prefer to have a password that's easy to type.